openmetadata-ui/src/main/resources/ui/src/components/MlModelDetail/SourceList.component.tsx

```diff
@@ -0,0 +1,89 @@
+/*
+ *  Copyright 2022 Collate
+ *  Licensed under the Apache License, Version 2.0 (the "License");
+ *  you may not use this file except in compliance with the License.
+ *  You may obtain a copy of the License at
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *  Unless required by applicable law or agreed to in writing, software
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome';
+import { Col, Row, Space, Typography } from 'antd';
+import React, { useMemo, useState } from 'react';
+import { useTranslation } from 'react-i18next';
+import { Link } from 'react-router-dom';
+import { EntityType } from '../../enums/entity.enum';
+import { MlFeature } from '../../generated/entity/data/mlmodel';
+import { getEntityLink } from '../../utils/TableUtils';
+import './SourceList.style.less';
+
+const SourceList = ({ feature }: { feature: MlFeature }) => {
```

Comment:
I am not sure if you did attach a recording for the same, Can you help me to identify it in screenshots or recordings?
P.S.: **Also, DO NOT CLUB MULTIPLE ISSUE ON SINGLE PR & IF YOU DO PLEASE USE RELEVANT PR TITLE**

Reply:
I've added a separate recording for the SourceList Component after the MLModel recording. Please do check.